lib/docs/scrapers/tailwindcss.rb

```diff
@@ -0,0 +1,37 @@
+module Docs
+  class Tailwindcss < UrlScraper
+    self.name = 'Tailwind CSS'
+    self.type = 'tailwindcss'
+    self.slug = 'tailwindcss'
+    self.base_url = 'https://tailwindcss.com/docs'
+    self.root_path = '/'
+    self.release = '2.0.3'
+
+    html_filters.push 'tailwindcss/entries', 'tailwindcss/clean_html'
+
+    # options[:container] = 'body';
+
+    options[:skip_patterns] = [
+
+        # removed because it focuses on how to use Tailwind with some other niche
+        # technologies and to also align with DevDoc's Vision, which is" to:"
+        # "indexing only the minimum useful to most developers" that use Tailwind
+        %r{\/guides\/.*},
+
+        # removed so it is easy to "get_type" (see tailwindcss/entries.rb line #15)
+        %r{\/colors\z}
+
+      ]
+
+    #Obtainable from https://github.com/tailwindlabs/tailwindcss/blob/master/LICENSE
+    options[:attribution] = <<-HTML
+      &copy; Adam Wathan, Jonathan Reinink
+      Licensed under the MIT License.
+    HTML
```

Comment:
The license footer is lacking a space before "licensed": 
![Screenshot_2021-05-01 Tailwind CSS saturate — DevDocs](https://user-images.githubusercontent.com/782446/116795149-20cc1880-aad3-11eb-8276-477672095ecb.png)